<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>findmedi</class>
 <widget class="QDialog" name="findmedi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>744</width>
    <height>486</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>744</width>
    <height>486</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>744</width>
    <height>486</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>查询药品信息</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>10</y>
     <width>271</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 26pt &quot;方正舒体&quot;;</string>
   </property>
   <property name="text">
    <string>查询药品信息</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>80</y>
     <width>741</width>
     <height>401</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,6,1">
      <item>
       <widget class="QComboBox" name="comboBox">
        <item>
         <property name="text">
          <string>请选择查询方式</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>按编号查询</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>按名称查询</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>按服用方法查询</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>按功效查询</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit"/>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>查询</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableView" name="tableView"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
